src/components/PageTitle.tsx

```diff
@@ -61,7 +61,8 @@ export const PageTitle: React.FC<Props> = ({
 const BadgeWrapper = styled.div<TitleTypographyProps>`
   display: inline-flex;
   margin-left: ${({ size }) => (size === "large" ? tokenSize.m : tokenSize.s)};
-  vertical-align: text-bottom;
+  vertical-align: ${({ size }) =>
+    size === "large" ? "middle" : "text-bottom"};
```

Comment:
Since this affects a bunch of pages can you spot check this to make sure nothing looks weird on the other pages its used.

Reply:
The only page that uses `size = large` is a Host page. So every other page that uses `PageTitle` should look exactly the same as deployed Spruce.